### ISSUE TITLE
Update README to document HTML generation best practices?

### ISSUE THREAD

**Issue by bashbaug** [KhronosGroup/SPIRV-Registry#181]
I think it would be a good idea to describe best practices when generating the HTML artifacts for an asciidoc extension.

I've been using something like the following, which matches the HTML output for many (but not all) of the HTML extensions in this repo:

```sh
$ asciidoc -b html5 -a icons -a toc2 -a toclevels=1 -o SPV_my_extension.html SPV_my_extension.asciidoc
```

There is probably an equivalent that uses the more modern asciidoctor toolchain but I haven't had the need to figure out what it is... yet.

Would this be helpful?  Should we document this in the README or using some other mechanism?  Should we include a Makefile in this repo?

**Comment by dgkoch** [comment]
IMO we should include a Makefile in this repo (and probably document it as well, but definitely a Makefile so that the html files can be (re-)generated in place).

**Comment by bashbaug** [comment]
Fixed by #189.
